### DISCUSSION TITLE
Problem connecting VSCODE with linux server: Could not establish connection to "2001"

### DISCUSSION THREAD

**Discussion by Goal1803** [neuefische/questions#838] [Web: Self-directed learning]
I have been tried to add an SSH connection from vscode and it keep saying Could 'not establish connection to "2001"'

I have tried many ways that I found through google but it doesn't work. Can someone please help?

**Comment by F-Kirchhoff** [comment]
Hey,
can you please specify on which operating system you are on and if you did our git and github setup? And could you please execute the following command inside your project folder in the terminal: 
```
git remote -v
```
Please post the response of this comment here.
